### PR TITLE
Squad request status notifications

### PR DESCRIPTION
**Pull request by ARtoriouSs**

### CODE REVIEW

File: priv/gettext/ru/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -2,6 +2,14 @@
 msgid "You are on duty tomorrow!"
 msgstr "Завтра ты назначен ответственным за лопату!"
 
+# squad_request approve notification text
+msgid "Your request to squad %{squad_number} was approved!"
+msgstr "Ваш запрос во взвод %{squad_number} был прият!"
```
Comment (mirterious) [inline priv/gettext/ru/LC_MESSAGES/default.po]: Typo

File: priv/gettext/ru/LC_MESSAGES/default.po [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+# squad_request rejection notification text
+msgid "Your request to squad %{squad_number} was rejected! Try to contact your squad commander"
+msgstr "Ваш запрос во взвод %{squad_number} был отклонён! Попробуйте связаться с командиром взвода"
```
Comment (mirterious) [inline priv/gettext/ru/LC_MESSAGES/default.po]: I suggest "Свяжитесь с командиром взвода". What do you think about it?

File: spec/domain/formations_spec.exs [hunk 3]
```diff
@@ -104,21 +104,49 @@ defmodule Squadster.Domain.FormationsSpec do
...
+    end
+
+    context "when user has enough permissions" do
+      it "deletes existing squad_request" do
+        previous_count = entities_count(SquadRequest)
+        Formations.delete_squad_request(squad_request().id, user())
+        expect entities_count(SquadRequest) |> to(eq previous_count - 1)
+      end
+
+      it "returns deleted squad_request" do
```
Comment (mirterious) [inline spec/domain/formations_spec.exs:120]: Do we really need this?

**Review by mirterious — CHANGES_REQUESTED** [review]

File: spec/domain/formations_spec.exs (hunk 3 above)
Comment (ARtoriouSs) [reply spec/domain/formations_spec.exs:120]: Why not  ¯\\_(ツ)_/¯

Comment (mirterious) [reply spec/domain/formations_spec.exs:120]: Make sense
image

**Review by mirterious — APPROVED** [review]

**Pull request merged**